### DISCUSSION TITLE
Redis ACL Requirements (external Redis server)

### DISCUSSION THREAD

**Discussion by michael1800v1** [goauthentik/authentik#11381] [Q&A]
I can't seem to find in the documents anywhere, including developer sections, what the access requirements are for the Redis user (Redis server-side user ACL). Unfortunately, initial connections on startup don't use the supplied username (though following requests do), so I had to give it @connection. Now it's terminating connections and boot-looping because of -@dangerous. 

While I can certainly debug output/traffic, I'm sure to run into additional command issues later on down the line as I add flows and providers. Does anyone know what AuthentiK's Redis user actually needs to fully function? I'd be happy with a link to documentation as well, but Google, repo searches and AuthentiK's site haven't provided anything along these lines.

**Comment by technicallymarv** [comment]
Not sure if you still need an answer to this almost a year after the fact. However, after going back and forth with it for a few hours I gave up and went with the following. Seems to allow it to do everything it needs while still maintaining user/pass requirements and removing access to dangerous commands:

+@all -@dangerous -flushdb -flushall -config -debug +info +slowlog

**Comment by rissson** [comment]
Btw, we're planning to remove redis with 2025.10
